test/runtests.py

```diff
@@ -18,14 +19,14 @@
 Samples:
 
 test all folders:
-    {0}
+    runtests.py
 
 test only Array:
-    {0} Array
+    runtests.py Array
 
 test a single file:
-    {0} Basics/hello.js
-'''.format(sys.argv[0]))
+    runtests.py Basics/hello.js
```

Comment:
When running from root folder, help was showing `test/runtests.py Basics/hello.js` however it should be `tests/runtests.py test/Basics/hello.js` .. simplified the help output instead.